libs/pulumi-lambda/src/lib/lambda-function.ts

```diff
@@ -103,5 +122,18 @@ export class LambdaFunction extends pulumi.ComponentResource {
                 dependsOn: [this.logGroup],
             },
         )
+
+        if (args.monitoring?.enabled) {
```

Comment:
IMO LambdaFunction shouldn't expose this, instead just `new MetricsAlarms` if you want alerms?